### PR TITLE
Unnest the fidesctl python project

### LINKED ISSUE #483: Version isn't properly reported in the Docker container
### Bug Description

Fidesctl thinks it is version 0 in Docker

#### Steps to Reproduce

1. Pull the docker image
2. open a shell
3. `fidesctl --version`

### Expected behavior

It should show the correct version

### Screenshots

_If applicable, add screenshots to help explain your problem._

### Environment

* Version:
* OS:
* Python Version:
* Docker Version:

### Additional context

This is caused by the root git file not being available at the build time of the image

### PR DESCRIPTION
**Pull request by ThomasLaPiana**
Closes #483 

### Code Changes

* [x] move the `fidesctl` dir contents up one level
* [x] update the .dockerignore file

### Steps to Confirm

* [x] running things locally works
* [x] `make build-local-prod ; docker run -it ethyca/fidesctl:local /bin/bash` -> `fidesctl --version` works
* [x] `make build-local-prod ; docker run -it ethyca/fidesctl:local /bin/bash` -> `fidesctl --local parse` works
* [x] `make docs-server` works

### Pre-Merge Checklist

* [x] All CI Pipelines Succeeded
* [x] Documentation Updated
* [x] Issue Requirements are Met
* [x] Relevant Follow-Up Issues Created

### Description Of Changes

Given that it seems we will not be adopting any sort of monorepo structure, the nesting of the actual python part of `fidesctl` no longer makes sense and is actually somewhat of a technical hurdle (specifically with how we're handling versioning). This PR un-nests `fidesctl` and turns `fides` into the proper home of the `fidesctl` python project.

### CODE REVIEW

**Comment by ThomasLaPiana** [comment]
everything is looking good, but need to make sure the docs are updated too. working on that now

**Comment by ThomasLaPiana** [comment]
the easiest way to test this is going to just be to pull the repo locally and take a look...the changelog is kinda useless here

File: .github/workflows/pr_checks.yml [hunk 1]
```diff
@@ -4,9 +4,6 @@ on:
   pull_request:
     branches:
       - main
-    paths:
-      - fidesctl/**
-      - .github/workflows/pr_checks.yaml
```
Comment (PSalant726) [inline .github/workflows/pr_checks.yml:9]: Did you intend to remove this completely? Should we instead just update it to exclude specific paths?

**Review by PSalant726 — CHANGES_REQUESTED** [review]
The changes look ok, but there's a seemingly fundamental issue somewhere:
```
root@0b0cbdef6529:/fides# fidesctl status
Traceback (most recent call last):
  File "/usr/local/bin/fidesctl", line 33, in <module>
    sys.exit(load_entry_point('fidesctl', 'console_scripts', 'fidesctl')())
  File "/usr/local/bin/fidesctl", line 22, in importlib_load_entry_point
    for entry_point in distribution(dist_name).entry_points
  File "/usr/local/lib/python3.8/importlib/metadata.py", line 503, in distribution
    return Distribution.from_name(distribution_name)
  File "/usr/local/lib/python3.8/importlib/metadata.py", line 177, in from_name
    raise PackageNotFoundError(name)
importlib.metadata.PackageNotFoundError: fidesctl
```

File: .github/workflows/pr_checks.yml (hunk 1 above)
Comment (ThomasLaPiana) [reply .github/workflows/pr_checks.yml:9]: yeah I meant to remove it. I'd rather bias to too much testing than too little. we can start exempting stuff if we want but the majority of the time we'd want testing anyway so 🤷

**Comment by ThomasLaPiana** [comment]
@PSalant726 the egg entry point didn't copy over for some reason, but I believe I've got it fixed, go ahead and try again

**Review by PSalant726 — APPROVED** [review]
I retested and everything looks good! 😅

**Comment by ThomasLaPiana** [comment]
> I retested and everything looks good! 😅

ty! did i win the award for most files touched 😂

**Pull request merged**